dogfooding/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: Flutter Dogfooding App to showcase Video SDK.
 
 environment:
   sdk: ">=3.4.0 <4.0.0"
-  flutter: ">=3.22.0"
+  flutter: ">=3.24.5"
```

Comment:
the new `color.withValues` is not available in the latest `3.27` so maybe set it as the min version for dogfooding.

Reply:
You are right, reverted to `withOpacity()`